README.md

```diff
@@ -35,16 +35,20 @@
 </div>  
 <br>
 
-`bpmn-visualization` is a TypeScript library for visualizing process execution data on [BPMN](https://www.omg.org/spec/BPMN/2.0.2/) diagrams with:
-- additional display options for execution data (highlight some transitions, counters, and more)
-- interactive capacities (mouse hover, click)
+`bpmn-visualization` is a TypeScript library for visualizing processes, in the [BPMN diagram](https://www.omg.org/spec/BPMN/2.0.2/) format, with simplicity.
```

Comment:
⚠️ let's talk about this introduction together as changing it as a large impact. If we do it, the user documentation, the repository description and the website MUST be updated as well.